### PR TITLE
:lipstick: Theme Retouch

### PR DESCRIPTION
**Pull request by prank1123**
# Theme-Retouch to make it more planet like...

iLugDelhi

### Changes: 
* changed the colour scheme of planet

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -13,20 +14,25 @@ module.exports = {
...
-    // },
+    colors: {
+      ...colors,
+      apnaGreen: "#04D98B",
+      dateGreen: "#038C5A",
+      apniHeading: "#014029",
+      neechuBG: "#3B3C40",
+      neechuText: "#f2f2f2",
+      ivory: "#FFFFF0",
+    },
```
Comment (AniVerma17) [inline tailwind.config.js:25]: Please rename the color keys to more semantic names.

File: tailwind.config.js [hunk 2]
```diff
@@ -13,20 +14,25 @@ module.exports = {
...
     fontFamily: {
       sans: [
         "system-ui",
         "sans-serif",
       ],
     },
+    boxShadow: {
+      ...defaultTailwind.boxShadow,
+      apnaShadow: "0 25px 50px -12px rgba(1, 64, 41, 0.25)"
+    }
```
Comment (AniVerma17) [inline tailwind.config.js:35]: Please use a more semantic name for custom shadows.

**Review by AniVerma17 — CHANGES_REQUESTED** [review]

**Comment by AniVerma17** [comment]
> # Theme-Retouch to make it more planet like...
> iLugDelhi
> 
> ### Changes:
> * changed the colour scheme of planet

@insaaniManav @hellozee @xeon-zolt need your feedback on the UX of this color scheme.

**Comment by AniVerma17** [comment]
@prank1123 please resolve the merge conflicts.

File: tailwind.config.js (hunk 1 above)
Comment (prank1123) [reply tailwind.config.js:29]: https://github.com/ILUGD/planet/pull/17/commits/6d34aa5839f0d39394d8fc15a7ca7b6301b04337 - resolved here

File: tailwind.config.js (hunk 2 above)
Comment (prank1123) [reply tailwind.config.js:39]: https://github.com/ILUGD/planet/pull/17/commits/9343d8e163eec3d570f14df551aea0260a214137 - fixed here

**Review by AniVerma17 — CHANGES_REQUESTED** [review]
@prank1123 your color scheme is not configured for dark mode. Please check the comments and make changes accordingly.

**Comment by prank1123** [comment]
> @prank1123 your color scheme is not configured for dark mode. Please check the comments and make changes accordingly.

Done as requested.

ilugd

File: src/index.njk [hunk 3]
```diff
@@ -5,11 +5,11 @@ layout: base.njk
 {% from 'macros/cards.njk' import feedCard %}
 {% include 'partials/banner.njk' %}
 <main>
-  <div class="mx-auto w-full bg-gray-300 items-center">
-    <div class="py-10 px-5 md:px-20 bg-gray-300 dark:bg-gray-800">
+  <div class="mx-auto w-full items-center">
+    <div class="py-10 px-5 md:px-20 bg-ivory dark:bg-darkGray">
```
Comment (AniVerma17) [inline src/index.njk:9]: Please apply a color class different from that of card sections for light mode, to maintain visual distinction between page background and content cards when the site UI is in light mode.

File: src/index.njk [hunk 4]
```diff
@@ -5,11 +5,11 @@ layout: base.njk
...
        <div class="mx-auto max-w-screen-xl grid sm:grid-cols-1 md:grid-cols-1 lg:grid-cols-3 grid-rows-1 gap-4 justify-items-center">
       {% asyncEach feed in collections.feeds %}
-        <section class="w-full py-4 px-4 m-2 shadow-md dark:shadow-md-light h-full bg-white dark:bg-gray-900 rounded-md">
+        <section class="w-full py-4 px-4 m-2 shadow-md h-full bg-ivory dark:bg-cardDarkGray rounded-md hover:shadow-cardShadow">
```
Comment (AniVerma17) [inline src/index.njk:12]: Please add appropriate shadow classes for dark mode as well.

**Review by AniVerma17 — CHANGES_REQUESTED** [review]

File: src/index.njk (hunk 4 above)
Comment (prank1123) [reply src/index.njk]: added border instead

**Comment by prank1123** [comment]
screencapture-8080-white-ptarmigan-cl3l6drl-ws-us13-gitpod-io-2021-08-08-19_16_52
screencapture-8080-white-ptarmigan-cl3l6drl-ws-us13-gitpod-io-2021-08-08-19_17_52

**Comment by AniVerma17** [comment]
IMO these borders look a bit weird, especially in dark mode. Try to implement shadows for both the modes.
Also, I suggest to use a different color (preferably a shade of gray) for card background in light mode.

**Comment by AniVerma17** [comment]
@prank1123 any update on this?

**Comment by AniVerma17** [comment]
Alright, closing this PR now since there hasn't been an update from the author on the last review for long.

**Pull request closed without merging**